crates/image_viewer/src/image_viewer.rs

```diff
@@ -73,15 +73,23 @@ impl Item for ImageView {
     fn tab_content(
         &self,
         _detail: Option<usize>,
-        _selected: bool,
+        selected: bool,
         _cx: &WindowContext,
     ) -> AnyElement {
-        self.path
+        let title = self
+            .path
             .file_name()
             .unwrap_or_else(|| self.path.as_os_str())
             .to_string_lossy()
-            .to_string()
-            .into_any_element()
+            .to_string();
+        h_flex()
```

Comment:
We shouldn't need this `h_flex` here. We can just render the `Label` directly.

Reply:
Thanks！